### PR TITLE
Methods - Generative Model

### PR DESCRIPTION
**Pull request by danich1**
This section describes the generative model used by this project. Did the best I could describing this model, but do think there might be some weak points in the description.

### CODE REVIEW

File: content/03.methods.md [hunk 1]
```diff
@@ -10,7 +10,30 @@ describe what a label function is and how many we created for each relation
 
 ## Training Models
 ### Generative Model
-talk about generative model and how it works
+The generative model is a core part of this automatic annotation framework.
+This model incorporates multiple signals emitted by label functions and assigns the most appropriate training class to each candidate sentence.
```
Comment (cgreene) [inline content/03.methods.md]:
```suggestion
It integrates multiple signals emitted by label functions and assigns a training class to each candidate sentence.
```

File: content/03.methods.md [hunk 2]
```diff
@@ -10,7 +10,30 @@ describe what a label function is and how many we created for each relation
...
+
+where $m$ is the number of candidate sentences,$F$ is the vector of summary statistics and $\theta$ is a vector of weights for each summary statistic.
+The summary statistics used by the generative model are as follows:  
+
+$$F^{Lab}_{i,j}(\Lambda, Y) = \unicode{x1D7D9}\{\Lambda_{i,j} \neq 0\}$$
+$$F^{Acc}_{i,j}(\Lambda, Y) = \unicode{x1D7D9}\{\Lambda_{i,j} = y_{i,j}\}$$   
+
+Where *Lab* stands for the label function propensity, which is the frequency of a label function emitting a signal.
+*Acc* is the individual label function's accuracy given the training class.
+This model finds the opimial weights ($\theta$) by minimizing the negative log likelihood:
```
Comment (cgreene) [inline content/03.methods.md]:
```suggestion
This model optimizes the weights ($\theta$) by minimizing the negative log likelihood:
```

File: content/03.methods.md [hunk 3]
```diff
@@ -10,7 +10,30 @@ describe what a label function is and how many we created for each relation
...
+{\sum_{\Lambda'}\sum_{y'} \exp(\sum_{i=1}^{m} \theta^{T}F_{i}(\Lambda', y'))}
+$$  
+
+where $m$ is the number of candidate sentences,$F$ is the vector of summary statistics and $\theta$ is a vector of weights for each summary statistic.
+The summary statistics used by the generative model are as follows:  
+
+$$F^{Lab}_{i,j}(\Lambda, Y) = \unicode{x1D7D9}\{\Lambda_{i,j} \neq 0\}$$
+$$F^{Acc}_{i,j}(\Lambda, Y) = \unicode{x1D7D9}\{\Lambda_{i,j} = y_{i,j}\}$$   
+
+Where *Lab* stands for the label function propensity, which is the frequency of a label function emitting a signal.
```
Comment (cgreene) [inline content/03.methods.md]:
```suggestion
*Lab* is the label function's propensity (the frequency of a label function emitting a signal).
```

File: content/03.methods.md [hunk 4]
```diff
@@ -10,7 +10,30 @@ describe what a label function is and how many we created for each relation
...
+*Acc* is the individual label function's accuracy given the training class.
+This model finds the opimial weights ($\theta$) by minimizing the negative log likelihood:
+
+$$\hat{\theta} = argmin_{\theta} -\sum_{\Lambda} log \sum_{Y}P(\Lambda, Y)$$
+
+Once optimized, the predictions from the generative model, $\hat{Y} = P(Y \mid \Lambda)$, are used as training classes for our dataset [@doi:10.1145/3209889.3209898; @doi:10.14778/3157794.3157797].
```
Comment (cgreene) [inline content/03.methods.md]:
```suggestion
In the framework we apply, predictions from the generative model, $\hat{Y} = P(Y \mid \Lambda)$ provide training classes for our dataset [@doi:10.1145/3209889.3209898; @doi:10.14778/3157794.3157797].
```

**Review by cgreene — APPROVED** [review]
When @dhimmel approves, feel free to merge.

**Review by dhimmel — APPROVED** [review]
Looks good to me.

**Pull request merged**